src/classes/VideoTimestamp.ts

```diff
@@ -83,4 +64,30 @@ export default class VideoTimestamp {
 
     return `${hours}h${minutes}m${seconds}s`;
   }
+
+  /**
+   * Converts a given timestamp string to seconds
+   *
+   * @param timestamp The timestamp to convert
+   *
+   * @return  The time in seconds
+   */
+  private static timestampToSeconds(timestamp: string): number {
+    let count = 0;
+
+    // eslint-disable-next-line security/detect-unsafe-regex
+    const match = timestamp.match(/^(\d+h)?(\d+m)?(\d+s)?$/);
```

Comment:
need to improve on this, shouldn't comment this out. Probably should break this up into different methods? yuck